server/graphql/v2/input/NewPaymentMethodInput.js

```diff
@@ -0,0 +1,14 @@
+import { GraphQLInputObjectType, GraphQLNonNull, GraphQLString } from 'graphql';
+
+import { NewPaymentMethodDataInput } from './NewPaymentMethodDataInput';
+
+export const NewPaymentMethodInput = new GraphQLInputObjectType({
+  name: 'NewPaymentMethodInput',
+  fields: () => ({
+    data: { type: new GraphQLNonNull(NewPaymentMethodDataInput) },
+    name: { type: new GraphQLNonNull(GraphQLString) },
+    service: { type: new GraphQLNonNull(GraphQLString) },
```

Comment:
Enum?

Reply:
What are the options besides 'stripe' for service?

Reply:
@znarf providing list (creditcard, banktransfer, prepaid, collectivebalance, giftcard, etc)